app/user/user.controller.js

```diff
@@ -1,13 +1,14 @@
 const express = require('express');
 const router = express.Router();
 const userService = require('./user.service');
+const _ = require('lodash');
 
 router.post('/authenticate', authenticate);
 router.get('/', findUsers);
-router.post('/', createUser);
-router.get('/:id', findUser);
-router.put('/:id', updateUser);
-router.delete('/:id', deleteUser);
+router.post('/register', createUser);
+router.get('/me', findUser);
```

Comment:
A co jeżeli wprowadzimy konto admina i będziemy chcieli przeglądać konto po id? `/me` to nie jest dobry pomysł